scripts/core/editor3/components/invisibles/SpaceComponent.jsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
+import {get} from 'lodash';
 import PropTypes from 'prop-types';
 
 export const SpaceComponent = (props) => {
     const spanProps = {...props};
+    const styleSet = get(props, 'children.0.props.styleSet');
+    let className = 'space-component';
+
+    if (styleSet) {
+        styleSet.forEach((style) => {
+            if (style.startsWith('DELETE_SUGGESTION')) {
```

Comment:
Don't you need to check if there is already the same class added?

Reply:
what do you mean?

Reply:
You are adding the className in the loop, which means you might add the same class multiple times. I think it might be better to avoid `forEach` altogether here

```
const deleteSuggestionExists = styleSet.filter((style) => style.startsWith('DELETE_SUGGESTION'));
const addSuggestionExists = styleSet.filter((style) => style.startsWith('ADD_SUGGESTION'));

if (deleteSuggestionExists) {
    className += ' delete-suggestion';
} else if (addSuggestionExists) {
    className += ' add-suggestion';
}
```

Reply:
I see, I don't think there can be more than one of these suggestions at a time, but it won't hurt to be sure